### PR TITLE
[Enhance] Benchmark PN++ on ScanNet and S3DIS

### PR DESCRIPTION
**Pull request by Wuziyi616**
We provide several settings on two datasets with BEST performance among original paper and third-party implementation.

### CODE REVIEW

**Comment by Wuziyi616** [comment]
Stay tuning. Can be finished within this week I think.

**Comment by Wuziyi616** [comment]
Stay tuning. Can be finished within this week I think.

File: configs/pointnet2/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+@inproceedings{qi2017pointnet++,
+  title={PointNet++ deep hierarchical feature learning on point sets in a metric space},
+  author={Qi, Charles R and Yi, Li and Su, Hao and Guibas, Leonidas J},
+  booktitle={Proceedings of the 31st International Conference on Neural Information Processing Systems},
+  pages={5105--5114},
+  year={2017}
+}
+```
+
+**Notice**: The original PointNet++ paper used step learning rate schedule. We discovered that cosine schedule achieves much better results and adopt it in our implementations. We also use a larger `weight_decay` factor because we find it consistently improving the performance.
```
Comment (ZwwWayne) [inline configs/pointnet2/README.md:19]: improves

File: configs/pointnet2/pointnet2_ssg_only_xyz_16x2_cosine_200e_scannet-3d-20class.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+        ])),
+    test_cfg=dict(
+        num_points=8192,
+        block_size=1.5,
+        sample_rate=0.5,
+        use_normalized_coord=False,
+        batch_size=24))
+
+# optimizer
+lr = 0.001  # max learning rate
```
Comment (ZwwWayne) [inline configs/pointnet2/pointnet2_ssg_only_xyz_16x2_cosine_200e_scannet-3d-20class.py:163]: we can create two basic schedules for segmentation in' _base_/schedules'

File: configs/pointnet2/README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## Results
+
+### ScanNet
+
+|        Method        |   Input   |   Lr schd   | Mem (GB) | Inf time (fps) | mIoU (Val set) | mIoU (Test set) | Download |
+| :------------------: |  :-----:  | :---------: | :------: | :------------: | :------------: | :------: |
+| [PointNet++ (SSG)](./pointnet2_ssg_only_xyz_16x2_cosine_200e_scannet-3d-20class.py) |    XYZ    | cosine 200e |    1.9    |           |           |           |[model]() &#124; [log]()|
+| [PointNet++ (SSG)](./pointnet2_ssg_16x2_cosine_200e_scannet-3d-20class.py) | XYZ+Color | cosine 200e |   1.9    |           |           |           |[model]() &#124; [log]()|
+| [PointNet++ (MSG)](./pointnet2_msg_only_xyz_16x2_cosine_200e_scannet-3d-20class.py) |    XYZ    | cosine 200e |   2.4    |           |           |           |[model]() &#124; [log]()|
```
Comment (ZwwWayne) [inline configs/pointnet2/README.md:29]: xyz-only might be better?

**Review by ZwwWayne — APPROVED** [review]
LGTM, can be merged after updating model links and resolving comments. Cheers!

File: configs/pointnet2/pointnet2_ssg_only_xyz_16x2_cosine_200e_scannet-3d-20class.py (hunk 2 above)
Comment (Wuziyi616) [reply configs/pointnet2/pointnet2_ssg_only_xyz_16x2_cosine_200e_scannet-3d-20class.py:163]: Fix.

File: configs/pointnet2/README.md (hunk 1 above)
Comment (Wuziyi616) [reply configs/pointnet2/README.md:19]: Fix.

File: configs/pointnet2/pointnet2_ssg_only_xyz_16x2_cosine_200e_scannet-3d-20class.py (hunk 2 above)
Comment (Wuziyi616) [reply configs/pointnet2/pointnet2_ssg_only_xyz_16x2_cosine_200e_scannet-3d-20class.py:163]: Good idea! Maybe one for ScanNet and one for S3DIS.

File: configs/pointnet2/README.md (hunk 3 above)
Comment (Wuziyi616) [reply configs/pointnet2/README.md]: Agree.

**Review by Wuziyi616 — COMMENTED** [review]
@ZwwWayne Update pre-trained model links. Ready for merge?

**Review by ZwwWayne — APPROVED** [review]
